src/app/instance.cc

```diff
@@ -176,15 +176,17 @@ void Instance::SetupTunnels() {
           ||tunnel.type == GetConfig().GetAttribute(Key::IRC)) {  // TODO(unassigned): see #9
         if (m_IsReloading) {
           auto client_tunnel = kovri::client::context.GetClientTunnel(tunnel.port);
-          if (client_tunnel && client_tunnel->GetName() != tunnel.name) {
-            // Conflicting port
-            // TODO(unassigned): what if we interchange two client tunnels' ports?
-            // TODO(unassigned): the addresses could differ
-            LOG(error)
-              << "Instance: " << tunnel.name
-              << " will not be updated, conflicting port";
-            continue;
-          }
+          if (client_tunnel && client_tunnel->GetName() != tunnel.name)
+            {
+              // Check for conflicting port done in ParseTunnelsConfig
+              // early deletion of client_tunnel to avoid temp duplicate port bind
```

Comment:
>ParseTunnelsConfig
>client_tunnel

FYI, in the future, I think it's best to avoid specifying names in comments because A) they can be missed during future refactoring B) IDEs may not catch them during automatic name-changing